### DISCUSSION TITLE
Selling a PCB

### DISCUSSION THREAD

**Discussion by malaki86** [nebhead/PiFire#155] [General]
I've been reading everything on the PiFire and I want one! The only problem for me is my soldering skills are nonexistent. I can build enclosures, run wiring,  etc. I can also do the software side. It's just the PCB itself that is my issue.

So, is anyone selling assembled PCBs, minus the Rpi? It would need the Rpi header, of course. I'd love for 4+ food probes (2 to start is fine).

**Comment by nebhead** [comment]
User Calonmerc on our discord was working on selling the PCB and components (possibly soldered?) so I'd recommend joining the discord and hitting him up on there.  I believe he was selling on Tindie, so search there as well.
